recipes/fuzzy-rough-learn/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "fuzzy-rough-learn" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ac850faf3397c531c80bca0b7ade0e0525a8f04714a915d4d54be519b44905fe
+
+build:
+  number: 0
+  skip: True  # [py<37]
```

Please remove the ``skip`` here, and I think this recipe could be ``noarch: python`` right?
With ``noarch`` you can just select which Python version you want to use
```suggestion
  noarch: python
```